src/components/Send.jsx

```diff
@@ -491,20 +499,31 @@ export default function Send() {
 
   return (
     <>
-      <div className="send">
+      <div
+        className={`send ${serviceInfo?.makerRunning && 'maker-running'} ${
+          serviceInfo?.coinjoinInProgress && 'taker-running'
+        }`}
+      >
         <PageTitle title={t('send.title')} subtitle={t('send.subtitle')} />
-
-        <rb.Fade in={!isCoinjoinOptionEnabled} mountOnEnter={true} unmountOnExit={true}>
-          <>
-            {serviceInfo?.makerRunning || serviceInfo?.coinjoinInProgress ? (
-              <div className="mb-4 p-3 border border-1 rounded">
-                <small className="text-secondary">
-                  {serviceInfo?.makerRunning && t('send.text_maker_running')}
-                  {serviceInfo?.coinjoinInProgress && t('send.text_coinjoin_already_running')}
-                </small>
-              </div>
-            ) : null}
-          </>
+        <rb.Fade in={isOperationDisabled()} mountOnEnter={true} unmountOnExit={true}>
+          <rb.Alert variant="info" className="mb-4">
+            <>
+              {serviceInfo?.makerRunning && (
+                <rb.Row>
+                  <rb.Col>
+                    <Trans i18nKey="send.text_maker_running">
+                      <Link to={routes.earn}>Earn</Link> is active. Stop the service in order to send collaborative
+                      transactions.
+                    </Trans>
+                  </rb.Col>
+                  <rb.Col xs="auto">
+                    <Sprite symbol="caret-right" width="24px" height="24px" />
```

Comment:
I feel like with the caret, the whole thing should be clickable not just "Earn" in the text. What do you think?

Reply:
Definitely!

Hmm.. trickier than expected.. I could not prevent browsers from underlining the text inside the alert with `a .alert { text-decoration: none }`.. That's why it's now done with an `.unstyled` class as workaround. : /

Maybe you have a cleaner solution at hand - that'd be great.

Reply:
Hm yeah I also couldn't get it to work right away. `unstyled` is fine imo.